packages/test/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solarwinds-apm/test",
-  "version": "0.0.0",
+  "version": "1.0.0-pre.1",
```

Comment:
Should this also be `1.0.0-pre.2`?

Reply:
This package didn't exist last time so no !

Reply:
The versions are currently pretty in sync but over time they might not be, which is voluntary

Reply:
Ah ok, got it!